ydb/library/yql/dq/runtime/dq_tasks_runner.cpp

```diff
@@ -779,10 +779,12 @@ class TDqTaskRunner : public IDqTaskRunner {
         return *ptr;
     }
 
-    std::pair<NUdf::TUnboxedValue, IDqAsyncInputBuffer::TPtr> GetInputTransform(ui64 inputIndex) override {
-        auto ptr = AllocatedHolder->InputTransforms.FindPtr(inputIndex);
-        YQL_ENSURE(ptr, "task: " << TaskId << " does not have input index: " << inputIndex << " or such transform");
-        return {ptr->TransformInput, ptr->TransformOutput};
+    std::optional<std::pair<NUdf::TUnboxedValue, IDqAsyncInputBuffer::TPtr>> GetInputTransform(ui64 inputIndex) override {
+        if (auto ptr = AllocatedHolder->InputTransforms.FindPtr(inputIndex)) {
+            return {std::pair{ptr->TransformInput, ptr->TransformOutput}};
```

Comment:
Why not create a struct with readble field names instead of a pair?

Reply:
For the consistency with the next method